shared/constants/types/chat2/message.js

```diff
@@ -139,14 +139,15 @@ export type _MessageAttachment = {
   outboxID: ?OutboxID,
   previewHeight: number,
   previewWidth: number,
+  previewTransferState: 'downloading' | null, // only for preview
   reactions: Reactions,
   submitState: null | 'deleting' | 'pending' | 'failed',
   timestamp: number,
   title: string,
   transferProgress: number, // 0-1 // only for the file
   transferState: 'uploading' | 'downloading' | 'remoteUploading' | null,
-  previewTransferState: 'downloading' | null, // only for preview
   type: 'attachment',
+  videoDuration: ?string,
```

Comment:
lets just type this as string and have it be '' if there's no duration, keeps the typing and checking simpler downstream